src/tests/test_CA/Makefile.am

```diff
@@ -118,6 +119,15 @@ softhsm2_one.conf:
 	@echo "objectstore.backend = file" >> $@
 	@echo "slots.removable = true" >> $@
 
+#Export cert from softhsm2 via p11tool, should produce the same as openssl
```

Comment:
I do not understand this comment. You use the *.der files to add the certificate to LDAP, how is this related to openssl?

Reply:
Initially I chased differences between openssl and gnutls(openssl ignorerd some settings while writing and reading certs that made everything produced by openssl work perfect)
I can remove half the comment since that problem solved itself.